src/app/tables/artifacts-table/artifacts-table.component.ts

```diff
@@ -80,6 +80,19 @@ export class ArtifactsTableComponent implements OnChanges {
         'See Story': this.getNodeStoryUrl(artifact)
       };
 
+      // If there are associated advisories, we can add links to those as well.
+      if (this.defaultColumns.indexOf('Advisories') > -1 && artifact.hasOwnProperty('advisories')) {
+        if (artifact.advisories[0]) {
+          const urls = {};
+          for (let i = 0; i < artifact.advisories.length; i++) {
+            const currentAdvisory = artifact.advisories[i];
+            const url = externalUrlPipe.transform(currentAdvisory);
+            urls[currentAdvisory['advisory_name']] = url;
+          }
+          this.linkColumnMappings[formattedArtifact[this.uidColumn]]['Advisories'] = urls;
```

Comment:
Would this still work for line 68, in the tableColumn pipe?

Reply:
What do you think about changing `linkColumnMappings` to the following format:
```javascript
{
    734506: {
        ID: {
            type: 'single',
            value: 'https://domain.local/something/734506'
        },
        Advisories: {
            type: 'multiple',
            value: {
                 'RHSA-123': 'https://domain.local/something/123',
                 'RHSA-124': 'https://domain.local/something/124'
            }
        }
    }
}
```

Then you could key on the `type` string in your `*ngIf` statements to determine how to display the value.

Reply:
I feel like there is no real clean solution for this :/ I think having a "type" key is still a bit confusing because now every time we make a link we will have to specify the type as well. But I also don't have a better idea than the way I have it now, so I am open to either doing this or leaving it. What are your thoughts?